packages/d/dotnet/package.yml

```diff
@@ -0,0 +1,82 @@
+name       : dotnet
+version    : 8.0.1
+release    : 1
+source     :
+    - git|https://github.com/dotnet/dotnet.git : b27976e5a6850466ee5b4ce24f91ee93bef645f7
+homepage   : https://dot.net/core
+license    : MIT
+component  : programming.dotnet
+summary    : .NET is a free, open-source, cross-platform framework for building modern apps and cloud services.
+description: |
+    .NET is a free, open-source, cross-platform framework for building modern apps and cloud services.
+networking : yes
+libsplit   : no
+clang      : no # current clang version breaks the build. We can try again when llvm is updated
+strip      : no
+builddeps  :
+    - pkgconfig(icu-i18n)
+    - pkgconfig(krb5)
+    - pkgconfig(libcurl)
+    - pkgconfig(libssl)
+    - pkgconfig(libunwind)
+    - pkgconfig(libxml-2.0)
+    - pkgconfig(lttng-ust)
+    - pkgconfig(zlib)
+    # - dotnet-sdk
+    # - dotnet-source-built-artifacts
+    - git
+    - lldb-devel
+    - ninja
+rundeps    :
+    - kerberos-devel
+    - libicu
+    - liblttng-ust
+    - libunwind-devel
+    - openssl-devel
+    - sdk:
+        - dotnet
+    - zlib-devel
```

Comment:
Are you sure that it needs the -devel versions of these libraries? That seems unlikely to me as the fedora version of this package doesn't seem to require it. Also, the non-devel versions would be picked up by autodep anyway (except for libicu which appears to be dl-opened)

Reply:
You're right. I did some digging and found the .net 8 [package requirements](https://github.com/dotnet/core/blob/main/release-notes/8.0/linux-packages.md) I've rebuild and run through the tests again with only `libicu` listed and it seemed to work perfectly fine.

I think it's preferable to leave `libicu` as a dep since the alternative is setting an environment variable or quietly break user applications

Reply:
Technically it's not possible to install a system without libicu as it's in system.base (as a dependency of libxml2 which is required by basically everything). However, I'd prefer to leave it as an explicit rundep just to be verbose here.